src/index.html

```diff
@@ -9,6 +9,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="wrapper">
+        <div class="header__logo">
+          <img src="images/logo.png" alt="MOYO logo">
+        </div>
```

Comment:
What about this requirement?
![image](https://user-images.githubusercontent.com/98764641/206685464-6b888afe-946b-4d12-adba-35d1fdf648f1.png)